src/DynamoCoreWpf/Rendering/HelixRenderPackage.cs

```diff
@@ -57,8 +57,99 @@ public HelixRenderPackage()
             lines = InitLineGeometry();
             mesh = InitMeshGeometry();
             lineStripVertexCounts = new List<int>();
+            Transform = System.Windows.Media.Media3D.Matrix3D.Identity.ToArray();
+        }
+
+        #endregion
+
+
+        #region ITransformable implementation
+
+        /// <summary>
+        /// A 4x4 matrix which is used to transform all geometry in the render packaage
+        /// </summary>
+        public double[] Transform { get; private set; }
+
+        /// <summary>
+        /// Sets the transform that will be applied to all geometry in the renderPackage
+        /// </summary>
+        /// <param name="transform"></param>
+        public void SetTransform(Autodesk.DesignScript.Geometry.CoordinateSystem transform)
+        {
+            var xaxis = transform.XAxis;
+            var yaxis = transform.YAxis;
+            var zaxis = transform.ZAxis;
+            var org = transform.Origin;
+
+            var csAsMat = new System.Windows.Media.Media3D.Matrix3D(xaxis.X, xaxis.Z, -xaxis.Y, 0,
```

Comment:
If we're going to negate these components, can we put a not here describing why?